frame/system/src/lib.rs

```diff
@@ -398,6 +398,15 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		#[pallet::weight(T::SystemWeightInfo::remark(_remark.len() as u32))]
+		pub fn remark_vec_u32(
+			origin: OriginFor<T>,
+			_remark: Vec<u32>,
+		) -> DispatchResultWithPostInfo {
```

Comment:
I believe that there may be a better solution to this, I will think about this, some ideas would be useful :)